src/sagemaker/transformer.py

```diff
@@ -97,6 +97,11 @@ def transform(self, data, data_type='S3Prefix', content_type=None, compression_t
             split_type (str): The record delimiter for the input object (default: 'None').
                 Valid values: 'None', 'Line', 'RecordIO', and 'TFRecord'.
             job_name (str): job name (default: None). If not specified, one will be generated.
+            input_filter (str): A JSONPath to select a portion of the input to pass to the algorithm container for
+                               inference. If you omit the field, it gets the value '$', representing the entire input. (default: None).
+            output_filter (str): A JSONPath to select a portion of the joined/original output to return as the output. (default: None).
```

Comment:
maybe add an example for the two filters?

Reply:
Added